charts/studio/values.yaml

```diff
@@ -432,10 +432,6 @@ studioBackend:
     targetCPUUtilizationPercentage: 80
     # targetMemoryUtilizationPercentage: 80
 
-  nameOverride: ""
-
-  fullnameOverride: ""
-
```

Comment:
Why did we remove `nameOverride` and `fullnameOverride`?

Reply:
@jesper7 Because it is not used. This particular one is in `studioBackend` block. I removed also from `studioUi` and `studioWorker`